src/Control/Session.php

```diff
@@ -167,6 +168,8 @@ public function __construct($data)
 
     /**
      * Init this session instance before usage
+     *
+     * @param HTTPRequest $request
```

Comment:
Phpdoc doesn't inherit from the method signature that way though. It needs to be explicitly provided, even though the php signature is equivalent. We rely on following the standard to build php docs. See psr 5 for the draft (and likely) standard.